app/models/service_plan.rb

```diff
@@ -4,4 +4,13 @@ class ServicePlan < ApplicationRecord
 
   belongs_to :portfolio_item
   validates :base, :presence => true
+  validate :modified_survey, :on => :update
+
+  private
+
+  def modified_survey
+    if Catalog::SurveyCompare.changed?(self)
+      raise Catalog::InvalidSurvey.new("Base survey does not match Topology")
```

Comment:
if you change this to `raise Catalog::InvalidSurvey, "Base survey does not match Topology"` it'll make the rubocop error go away while retaining functionality